samples/net/sockets/echo_server/boards/intel_socfpga_agilex5_socdk.conf

```diff
@@ -0,0 +1,24 @@
+# Copyright (c) 2022-2023, Intel Corporation.
+# SPDX-License-Identifier: Apache-2.0
+
+CONFIG_MDIO=y
+CONFIG_NET_L2_ETHERNET=y
+
+CONFIG_NET_PKT_RX_COUNT=64
+CONFIG_NET_PKT_TX_COUNT=64
+CONFIG_NET_BUF_RX_COUNT=1024
+CONFIG_NET_BUF_TX_COUNT=512
+
+CONFIG_ISR_STACK_SIZE=4096
+CONFIG_MAIN_STACK_SIZE=10240
+CONFIG_IDLE_STACK_SIZE=2048
+CONFIG_SHELL_STACK_SIZE=10240
+CONFIG_SYSTEM_WORKQUEUE_STACK_SIZE=8192
+CONFIG_NET_MGMT_EVENT_STACK_SIZE=8192
+CONFIG_NET_TCP_WORKQ_STACK_SIZE=4096
+CONFIG_NET_RX_STACK_SIZE=32768
+
+CONFIG_NET_CONFIG_MY_IPV4_ADDR="10.10.0.102"
+CONFIG_NET_CONFIG_MY_IPV4_NETMASK="255.255.255.0"
+CONFIG_NET_CONFIG_MY_IPV4_GW="8.8.8.8"
+CONFIG_NET_CONFIG_PEER_IPV4_ADDR="10.10.0.101"
```

Comment:
Please remove these.

Reply:
@jukkar, the values specified in our configuration file are provisional and have been modified to ensure the functionality of these applications on our Intel Simics® Simulator target. At present, our development kits are not accessible in the market. We plan to eliminate these values once we conduct tests on real hardware. Still, if it seems atypical or incorrect at this point for upstream integration, we will review and make necessary updates to these values.

Reply:
removed these configurations